### PR TITLE
Update dependencies 20190702

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,40 +42,40 @@
...
-    "chromedriver": "^75.0.0",
+    "chromedriver": "^75.0.1",
     "copy-webpack-plugin": "^5.0.3",
     "core-js": "3.1.4",
     "css-loader": "^3.0.0",
     "cucumber": "cucumber/cucumber-js#issue-727-retry",
     "cucumber-pretty": "^1.5.2",
     "easygettext": "^2.7.0",
-    "eslint": "5.16.0",
+    "eslint": "6.0.1",
```
Comment (LukasHirt) [inline package.json]: Build when running `yarn watch-all` still fails because of the eslint.

Comment (DeepDiver1975) [reply package.json]: let's nail eslint to < 6 - THX - will take care

Comment (DeepDiver1975) [reply package.json]: updated ...

Comment (LukasHirt) [reply package.json]: Works fine now, thanks!

**Pull request merged**